src/core/shape/vertex.js

```diff
@@ -651,9 +654,18 @@ p5.prototype.endContour = function() {
  * precision mediump float;
  *
  * out vec4 outColor;
+ * in int instanceID;
```

Comment:
Not 100% sure if this is why this example is throwing an error compiling the shader, but it looks like the vertex shader uses a lowercase d in `Id` while the fragment shader capitalizes it

Reply:
That's on reason why, but int's also can't be interpolated like that so ur also supposed to use `flat`.

Reply:
Ah sorry, that's my fault and what I get for editing the code inline in browser. Thanks for catching!